src/components/project_card.rs

```diff
@@ -0,0 +1,108 @@
+use yew::{html, Component, ComponentLink, Html, Properties, ShouldRender};
+
+#[derive(Clone, Copy,PartialEq, Eq, Hash)]
+pub enum Lang {
+    Rust,
+    CSS,
```

Comment:
Please remember to run `cargo clippy`, Rust doesn't make use of all caps identifiers